lib/datadog/tracing/trace_operation.rb

```diff
@@ -25,7 +25,7 @@ module Tracing
     #
     # @public_api
     class TraceOperation
-      include Metadata::Tagging
+      include Metadata
```

Comment:
sorry, I will have to block because of this change - I don't think we have to include both `Metadata::Errors` and `Metadata::Analytics` here just to require 'metadata/metastruct' file?